docs/reference/commandline/update.md

```diff
@@ -38,6 +39,10 @@ options on a running or a stopped container. You can only update
 stopped container, the next time you restart it, the container uses those
 values.
 
+Another configuration you can change with this command is restart policy,
+new restart policy will take effect instantly after you run `docker update`
+on a stopped or running container.
```

Comment:
can I `docker update` on a "created" container? If so, I think we can skip the "stopped or running"